salt/utils/schedule.py

```diff
@@ -385,7 +385,7 @@ class Schedule(object):
     '''
     instance = None
 
-    def __new__(cls, opts, functions, returners=None, intervals=None, cleanup=None, proxy=None):
+    def __new__(cls, opts, functions, returners=None, intervals=None, cleanup=None, proxy=None, isolate=None):
```

Comment:
I am wondering if the word `standalone` would be better than `isolate`. Thoughts?

Reply:
meh, I think you are correct....

Reply:
`standalone` is more accurate.  I'll update the PR.